src/engine/audio/index.ts

```diff
@@ -197,6 +189,29 @@ export const setAudioListener = (audioState: AudioState, eid: number) => {
 
 export const setAudioPeerEntity = (audioState: AudioState, peerId: string, eid: number) => {
   audioState.peerEntities.set(peerId, eid);
+
+  const mediaStreamSource = audioState.peerMediaStreamSourceMap.get(peerId);
+  if (!mediaStreamSource)
+    return console.error("could not setAudioPeerEntity - mediaStreamSource not found for peer", peerId);
+
+  const panner = audioState.entityPanners.get(eid);
+  if (!panner) return console.error("could not setAudioPeerEntity - panner not found for eid", eid, "peerId", peerId);
+
+  mediaStreamSource.connect(panner);
+};
+
+export const setPeerMediaStream = (audioState: AudioState, peerId: string, mediaStream: MediaStream) => {
+  if (window.navigator.userAgent.includes("Chrome")) {
```

Comment:
Maybe we should use `!!window.chrome` instead? We want to detect all Chromium based browsers, which I think that will do.